src/ui/public/saved_objects/saved_objects_client.js

```diff
@@ -0,0 +1,95 @@
+import { resolve as resolveUrl, format as formatUrl } from 'url';
+import { pick, partial } from 'lodash';
+
+import { SavedObject } from './saved_object';
+
+const join = (...uriComponents) => (
+  uriComponents.filter(Boolean).map(encodeURIComponent).join('/')
+);
+
+export class SavedObjectsClient {
+  constructor($http, basePath) {
+    this._$http = $http;
+    this._apiBaseUrl = `${basePath}/api/saved_objects/`;
+  }
+
+  get(type, id) {
+    if (!type || !id) {
+      return Promise.reject(new Error('requires type and id'));
```

Comment:
Good catch, swapped it out here: https://github.com/elastic/kibana/pull/11632/commits/5fa37cc8dcc050f70d8068109f7e2fd5e3f80a4f